### PR TITLE
Applied the orange theme to certain pages

### PR DESCRIPTION
**Pull request by AMMAR-62**
Some merge conflicts may occur

### CODE REVIEW

**Comment by AMMAR-62** [comment]
image
@classes
image
@Resources
image
@Cart

File: client/src/components/Product Page/Product_materials/CourseContents.js [hunk 1]
```diff
@@ -41,28 +42,30 @@ const CourseListStyles = makeStyles((theme) => ({
...
   ListSubBack: {
-    background: "#3949ab",
-    borderRadius: "10px",
-    boxShadow: "1px 1px #3949ab",
-    position: "static",
-    color: "white",
+    // background: "#f1f1f1",
+    // borderRadius: "10px",
+    // boxShadow: "1px 1px #3949ab",
+    // position: "static",
```
Comment (ShubhankarKG) [inline client/src/components/Product Page/Product_materials/CourseContents.js:51]: Please remove the comments

File: client/src/components/Product Page/Product_materials/CourseContents.js [hunk 2]
```diff
@@ -41,28 +42,30 @@ const CourseListStyles = makeStyles((theme) => ({
...
   },
   Enroll: {
-    background: "#3949ab",
-    color: "white",
+    background: "#f26522",
+    color: "#ffffff",
     margin: "1rem auto",
     borderRadius: "2rem",
-    border: "1px outset",
+    // border: "1px outset",
```
Comment (ShubhankarKG) [inline client/src/components/Product Page/Product_materials/CourseContents.js:68]: Please remove the comment

File: client/src/components/Product Page/Product_materials/CoursePageHeader.js [hunk 3]
```diff
@@ -9,9 +9,9 @@ const CoursePageHeaderStyles = makeStyles({
     justifyContent: "space-around",
     margin: "10px 10%",
     // background: "#90caf9",
-    background: "#42a5f4",
+    background: "#f1f1f1",
     borderRadius: "20px",
-    boxShadow: "4px 4px #1565c0",
+    // boxShadow: "4px 4px ",
```
Comment (ShubhankarKG) [inline client/src/components/Product Page/Product_materials/CoursePageHeader.js:14]: Please remove the comment

File: client/src/components/Product Page/Product_materials/CoursePageHeader.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ const CoursePageHeaderStyles = makeStyles({
     width: "50%",
     fontWeight: "400",
     margin: "auto",
-    color: "white",
+    // color: "#f26522",
```
Comment (ShubhankarKG) [inline client/src/components/Product Page/Product_materials/CoursePageHeader.js:26]: Please remove the comment

File: client/src/components/Product Page/Product_materials/CourseTypePane.js [hunk 5]
```diff
@@ -24,8 +20,8 @@ const mainPaneStyles = makeStyles({
     fontSize: "25px",
     display: "flex",
     justifyContent: "center",
-    color: "white",
-    boxShadow: "3px 3px #1c87e5"
+    color: "#f26522"
+    // boxShadow: "3px 3px"
```
Comment (ShubhankarKG) [inline client/src/components/Product Page/Product_materials/CourseTypePane.js:24]: Please remove the comment

File: client/src/components/Resource Page/ClassPane/ClassPane.js [hunk 6]
```diff
@@ -30,6 +45,7 @@ const ClassPane = ({ Course, CoursesList }) => {
         <ReactElasticCarousel
           breakPoints={breakPoints}
           className={classes.slider}
+          style={{}}
```
Comment (ShubhankarKG) [inline client/src/components/Resource Page/ClassPane/ClassPane.js:48]: Please remove the style if not needed

File: client/src/components/Resource Page/ClassPane/Product.js [hunk 7]
```diff
@@ -5,12 +5,12 @@ import img1 from "../../../assets/images/resourceImages/img1.jpeg";
...
   classHolder: {
-    border: "30px solid #90caf9",
-    borderRadius: "0px 20px",
     display: "flex",
     flexDirection: "column",
     justifyContent: "space-around",
-    boxSizing: "border-box"
+    padding: "1rem",
+    margin: "2px 0px"
+    // border:"2px",
```
Comment (ShubhankarKG) [inline client/src/components/Resource Page/ClassPane/Product.js:13]: Please remove the comment

File: client/src/components/Resource Page/ClassesPane.js [hunk 8]
```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import ClassPane from "./ClassPane/ClassPane";
 import { makeStyles } from "@material-ui/core";
+// import { Product } from "./ProductsFetcher";
```
Comment (ShubhankarKG) [inline client/src/components/Resource Page/ClassesPane.js:4]: Please remove the comment

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]
Thanks, only cleanup required.

Please re-add the `server/.mocharc.yml` file as that is needed.

**Comment by pal-16** [comment]
> Please re-add the `server/.mocharc.yml` file as that is needed.

Thanks for the changes. Just the above file needs to be added.

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**